File: CRS-GROUP-A-Angular-UI-UX-App/src/app/models/registered-course.ts
export class RegisteredCourse {
	
	private courseId: number;
	private courseName:string;
	private studentId: number;
	private registeredStatus: number;
	private grade: string;
	
	constructor(courseId: number, courseName:string, studentId: number, registeredStatus: number, grade: string) {
		this.courseId = courseId;
		this.courseName = courseName;
		this.studentId = studentId;
		this.registeredStatus = registeredStatus;
		this.grade = grade;
	}
	
	getCourseId(): number {
		return this.courseId;
	}

	setCourseId(courseId: number) {
		this.courseId = courseId;
	}

	getCourseName(): string {
		return this.courseName;
	}

	setCourseName(courseName: string) {
		this.courseName = courseName;
	}

	getStudentId(): number {
		return this.studentId;
	}

	setStudentId(studentId: number) {
		this.studentId = studentId;
	}

	getRegisteredStatus(): number {
		return this.registeredStatus;
	}

	setRegisteredStatus(registeredStatus: number) {
		this.registeredStatus = registeredStatus;
	}

	getGrade(): string {
		return this.grade;
	}

	setGrade(grade: string) {
		this.grade = grade;
	}	
}

File: CRS-Group-A-Student-Producer/src/main/java/com/lti/dto/RegisteredCourse.java
package com.lti.dto;

import java.io.Serializable;

import org.springframework.stereotype.Component;

@Component
public class RegisteredCourse implements Serializable {
	
	private int courseId;
	private String courseName;
	private int studentId;
	private int registeredStatus;
	private String grade;
	
	public RegisteredCourse() {}
	
	public RegisteredCourse(int courseId, String courseName, int studentId, int registeredStatus, String grade) {

		this.courseName = courseName;
		this.courseId = courseId;
		this.studentId = studentId;
		this.registeredStatus = registeredStatus;
		this.grade = grade;
	}
	
	/**
	 * @return the courseId
	 */
	public int getCourseId() {
		return courseId;
	}
	/**
	 * @param courseId the courseId to set
	 */
	public void setCourseId(int courseId) {
		this.courseId = courseId;
	}
	/**
	 * @return the studentId
	 */
	public int getStudentId() {
		return studentId;
	}
	/**
	 * @param studentId the studentId to set
	 */
	public void setStudentId(int studentId) {
		this.studentId = studentId;
	}
	/**
	 * @return the registeredStatus
	 */
	public int getRegisteredStatus() {
		return registeredStatus;
	}
	/**
	 * @param registeredStatus the registeredStatus to set
	 */
	public void setRegisteredStatus(int registeredStatus) {
		this.registeredStatus = registeredStatus;
	}
	/**
	 * @return the grade
	 */
	public String getGrade() {
		return grade;
	}
	/**
	 * @param grade the grade to set
	 */
	public void setGrade(String grade) {
		this.grade = grade;
	}

	/**
	 * @return the courseName
	 */
	public String getCourseName() {
		return courseName;
	}

	/**
	 * @param courseName the courseName to set
	 */
	public void setCourseName(String courseName) {
		this.courseName = courseName;
	}	
}

